src/Control/Session.php

```diff
@@ -139,6 +142,7 @@ class Session
      *
      * @see https://secure.php.net/manual/en/function.session-cache-limiter.php
      * @var string|null
+     * @deprecated 6.1.0 Will be removed without equivalent functionality to replace it
      */
     private static $sessionCacheLimiter = '';
```

Comment:
As discussed in the issue description, this can be used to set headers that _should not_ be tied to session.